frontend/src/app/services/state.service.ts

```diff
@@ -49,7 +49,7 @@ const defaultEnv: Env = {
   'BISQ_ENABLED': false,
   'BISQ_SEPARATE_BACKEND': false,
   'ITEMS_PER_PAGE': 10,
-  'KEEP_BLOCKS_AMOUNT': 8,
+  'KEEP_BLOCKS_AMOUNT': 22,
```

Comment:
if we want to show only 11 blocks then why keep 22? or should we keep more?

Reply:
22 is to show 11 blocks.

Reply:
can it not be 11 then?